### PR TITLE
KubeVirt: enable the seccomp feature gate and custom profile

### PR DESCRIPTION
**Pull request by jean-edouard**
**Reviewer Checklist**

> Reviewers are supposed to review the PR for every aspect below one by one. To check an item means the PR is either "OK" or "Not Applicable" in terms of that item. All items are supposed to be checked before merging a PR. 

- [ ] PR Message
- [ ] Commit Messages
- [ ] How to test
- [ ] Unit Tests
- [ ] Functional Tests
- [ ] User Documentation
- [ ] Developer Documentation
- [ ] Upgrade Scenario
- [ ] Uninstallation Scenario
- [ ] Backward Compatibility
- [ ] Troubleshooting Friendly
  
**Release note**:

```release-note
KubeVirt is now configured to always install and use its custom seccomp policy
```

### CODE REVIEW

**Review by acardace — APPROVED** [review]

File: controllers/hyperconverged/hyperconverged_controller_test.go [hunk 1]
```diff
@@ -223,6 +224,13 @@ var _ = Describe("HyperconvergedController", func() {
 				Expect(kv.Spec.Configuration.DeveloperConfiguration.FeatureGates).To(HaveLen(len(expectedFeatureGates)))
 				Expect(kv.Spec.Configuration.DeveloperConfiguration.FeatureGates).To(ContainElements(expectedFeatureGates))
 
+				// Ensure the KubeVirt seccomp profile is set
+				Expect(kv.Spec.Configuration.SeccompConfiguration).ToNot(BeNil())
+				Expect(kv.Spec.Configuration.SeccompConfiguration.VirtualMachineInstanceProfile).ToNot(BeNil())
+				Expect(kv.Spec.Configuration.SeccompConfiguration.VirtualMachineInstanceProfile.CustomProfile).ToNot(BeNil())
+				Expect(kv.Spec.Configuration.SeccompConfiguration.VirtualMachineInstanceProfile.CustomProfile.RuntimeDefaultProfile).To(BeFalse())
+				Expect(kv.Spec.Configuration.SeccompConfiguration.VirtualMachineInstanceProfile.CustomProfile.LocalhostProfile).To(Equal("kubevirt/kubevirt.json"))
```
Comment (nunnatsa) [inline controllers/hyperconverged/hyperconverged_controller_test.go:232]: This lines fails in the unit test.

Please fix:
```suggestion
				Expect(*kv.Spec.Configuration.SeccompConfiguration.VirtualMachineInstanceProfile.CustomProfile.LocalhostProfile).To(Equal("kubevirt/kubevirt.json"))
```

**Review by nunnatsa — CHANGES_REQUESTED** [review]
Please fix the unit test, so the sanity test will pass.

File: controllers/hyperconverged/hyperconverged_controller_test.go (hunk 1 above)
Comment (jean-edouard) [reply controllers/hyperconverged/hyperconverged_controller_test.go]: :facepalm: thank you! Fixed

**Review by tiraboschi — APPROVED** [review]

**Pull request merged**